test/parallel/test-stream-transform-callback-twice.js

```diff
@@ -7,7 +7,7 @@ const stream = new Transform({
 
 stream.on('error', common.expectsError({
   type: Error,
-  message: 'Callback called multiple times',
+  message: 'The undefined callback was called more than once',
```

Comment:
Should we not be updating the code that generates this error so that it does not have 'undefined' but an indication of what the callback is ?